src/pretix/base/models/event.py

```diff
@@ -330,6 +330,21 @@ def get_payment_providers(self) -> dict:
                 providers[pp.identifier] = pp
         return providers
 
+
+    @property
+    def get_event_microdata(self):
+        import json
+
+        eventdict = {"@context": "http://schema.org", "@type": "Event"}
+        eventdict["location"] = {"@type": "Place",
+                                 "address": str(self.location)}
+        eventdict["startDate"] = str(self.date_from)
```

Comment:
1. We should be specific about the dateformat. As I see it,  in this place ``self.date_from.isoformat()`` would be correct.
2. According to Google's documents, we should only specify the date if the time is not known. We have a setting for that:
````
if self.settings.show_times:
    eventdict["startDate"] = self.date_from.isoformat()
else:
    eventdict["startDate"] = self.date_from.date().isoformat()
```